app/src/main/java/com/example/scoutconcordia/Activities/MapsActivity.java

```diff
@@ -1195,6 +1208,26 @@ private void drawDirectionsPath(LatLng origin, LatLng dest){
         catch(Throwable t){
             Log.d(TAG, t.getMessage());
         }
+        int estimatedTime = new BigDecimal(time/60).setScale(0, RoundingMode.HALF_UP).intValue();
+        if(mode.toString().equals("driving") || mode.toString().equals("walking") || mode.toString().equals("transit")){
+//            Toast.makeText(MapsActivity.this, "estimated time in "+ estimatedTime + "mins",  Toast.LENGTH_LONG).show();
+            if (startingPoint.length() > 8 && startingPoint.substring(startingPoint.length() - 8).equals("Building")) //if the starting point is a building
+            {
+                if (destination.length() > 8 && destination.substring(destination.length() - 8).equals("Building")) //if the destination is a building
+                {
+//                    Toast.makeText(MapsActivity.this, "from:  " + startingPoint + "to: " + destination, Toast.LENGTH_LONG).show();
```

Comment:
Will we need this in the future? If not we can remove it

Reply:
we need these to make sure the user input is building